docs/semgrep-code/notifications.md

```diff
@@ -402,40 +403,146 @@ If you are using GitHub Actions to run Semgrep, no extra changes are needed to g
+:::
+
+Fulfill these general steps to create a repository access token:
+
+1. Create a repository access token in Bitbucket with **Read**, and **Write** permissions for pull requests. Follow the instructions in [Create a repository Access Token](https://support.atlassian.com/bitbucket-cloud/docs/create-a-repository-access-token/) in Bitbucket documentation.
+1. Add the repository access token as a repository variable with the **Secured** option.
+
+To complete the configuration, follow the [Adding Semgrep to your Bitbucket CI pipeline for PR comments](/semgrep-code/notifications/#bitbucket-ci-pipelines-yaml-file-for-pr-comments) section.
+
+#### Bitbucket CI pipelines YAML file for PR comments
+
+:::info Prerequisite
+Finish the steps in either [Creating a workspace access token](/semgrep-code/notifications/#creating-and-adding-a-workspace-access-token) (if you have **Bitbucket Cloud Premium** plan), or [Creating a repository access token](/semgrep-code/notifications/#creating-and-adding-a-repository-access-token) before committing the file below.
+:::
+
+Commit the following code to your `bitbucket-pipelines.yml` file:
+
+```yaml
+image: atlassian/default-image:latest
+
+pipelines:
+  branches:
+    main:
+        - step:
+            name: 'Run Semgrep full scan with main branch'
+            image: returntocorp/semgrep
+            script:
+              - export SEMGREP_APP_TOKEN=$SEMGREP_APP_TOKEN
+              - semgrep ci
+  pull-requests:
+    '**':
+      - step:
+          name: 'Run Semgrep diff scan with PR branch'
+          image: returntocorp/semgrep
+          script:
+            # Set SEMGREP Variables
+            - export SEMGREP_APP_TOKEN=$SEMGREP_APP_TOKEN
+            - export SEMGREP_BASELINE_REF="origin/main"
+            - git fetch origin "+refs/heads/*:refs/remotes/origin/*"
+            - export BITBUCKET_TOKEN=$PAT
+            - semgrep ci
+```
+Substitute the placeholder <code><span className="placeholder">PAT</span></code> and <code><span className="placeholder">main</span></code> according to your workspace or repository settings.
+
+:::info
+The updated `bitbucket-pipelines.yml` YAML file must be also included in the branches where you want Semgrep to run.
+:::
+
+As a result, your Bitbucket repositories are now part of the [Projects](https://semgrep.dev/orgs/-/projects) page. You may now receive comments on PRs from Semgrep Cloud Platform. For more configuration options, see [Bitbucket Pipelines CI Sample](/semgrep-ci/sample-ci-configs/#bitbucket-pipelines). Examples of more CI pipelines that you can use in Bitbucket CMS can be found in [Sample continuous integration (CI) configurations](https://semgrep.dev/docs/semgrep-ci/sample-ci-configs/).
+
+:::info
+- Test Semgrep PR comments by submitting a test code from a rule in your [Rule board](https://semgrep.dev/orgs/-/board) that is in the **Comment** column.
```

Comment:
```suggestion
```
This is not easy to do. They'd have to introduce a vulnerability. Suggest removing it.

Reply:
I agree.

Reply:
By not easy, I mean it's also not good practice to introduce something that violates a Semgrep rule just to test this out, as many Semgrep rules are security rules.

Reply:
@s-santillan do we have a better way to test this?

Reply:
@s-santillan I will apply your suggestion (and mine in the same text above) and we can add this later in another PR...